modules/developers-guide/examples/go/go-parameterized.go

```diff
@@ -0,0 +1,33 @@
+any, err := anypb.New(
+	&pb.Values{
+		Values: []*pb.Value{
```

Comment:
The indent here is not consistent with other places. I noticed it because it was much further over compared to other code blocks.